modules/library/render/src/test/java/org/geotools/renderer/lite/LabelSpacingTest.java

```diff
@@ -61,6 +61,6 @@ public void testLabelsIncreaseWordSpacing() throws Exception {
         // let's see if the result image match our expectations
         File reference = new File(
                 "./src/test/resources/org/geotools/renderer/lite/test-data/lineLabelsIncreaseWordSpacing.png");
-        ImageAssert.assertEquals(reference, image, 3000);
+        ImageAssert.assertEquals(reference, image, 3500);
```

Comment:
This difference seems considerable looking at the images below, it is like we are respecting kerning in Java 9. Okay it is only a change on text that follows a line, so it it a result of different math?

Should we note this as an improvement for those upgrading to Java 9....

Reply:
Yeah, some of the labels look like they didn't change at all, but one or two seem to have gotten shifted. I assume it is related to the math difference, but I can't corroborate it.

Given the already high tolerance on the test, I wouldn't be surprised if this is already somewhat different between different OSes and architectures - if we really wanted to be sure how it changed, we would have to test on multiple different OSes running Java 9 vs Java 8 (and possible OpenJDK vs Oracle)